CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## 1.50.0 - Nature
 
+- Improve speed of `tuist cache warm` by parallelizing the target cache checks [#3462](https://github.com/tuist/tuist/pull/3462) by [@bolismauro](https://github.com/bolismauro)
```

Comment:
Let's mention this is beneficial when using remote cache, I am assuming for local cache we won't get noticeable improvement because the bottleneck is the I/O. If you noticed improvement in both local and remote please mind share a benchmark 😄

Reply:
Right :D Changed accordingly